src/mscorlib/src/System/AppContext/AppContextDefaultValues.cs

```diff
@@ -26,6 +26,18 @@ public static void PopulateDefaultValues()
         [System.Security.SecuritySafeCritical]
         private static void ParseTargetFrameworkName(out string identifier, out string profile, out int version)
         {
+#if FEATURE_CORECLR
+            if (CompatibilitySwitches.IsAppSilverlight81)
+            {
+                // Since the SL does not have any Main() the reading of the TFM will not work and as a workaround we use the CompatibilitySwitch.IsAppSilverlight81 
```

Comment:
Please reword a little what is "the SL" :) also it would be good to wrap this line a little shorter as I had to scroll to read the full line.